src/common/gui/CScalableBitmap.cpp

```diff
@@ -40,17 +52,36 @@ CScalableBitmap::CScalableBitmap(CResourceDescription desc) : CBitmap(desc)
         }
         
     }
+    lastSeenZoom = currentPhysicalZoomFactor;
 }
 
+
 void CScalableBitmap::draw (CDrawContext* context, const CRect& rect, const CPoint& offset, float alpha )
 {
+    if (lastSeenZoom != currentPhysicalZoomFactor)
+    {
+        int ns = -1;
+        for (auto s : scales)
+        {
+            if (s>=currentPhysicalZoomFactor && ns<0)
+                ns = s;
+        }
+        if (ns<0)
+        {
+            ns = scales.back();
+        }
+        bestFitScaleGroup = ns;
+        lastSeenZoom = currentPhysicalZoomFactor;
+    }
     /*
     ** For now this is a fixed 'retina' style draw of using 2x bitmaps at 0.5 scale if they exist
     */
-    if (scaledBitmaps[ 200 ] != NULL)
+    if (scaledBitmaps[ bestFitScaleGroup ] != NULL)
```

Comment:
A nitpick: `scaledBitmaps[ bestFitScaleGroup ]`

Reply:
Isn't that what it says?

Reply:
Or do you mean kill the != NULL? Yeah you can go both ways.